src/DropdownStateMixin.js

```diff
@@ -56,7 +57,7 @@ const DropdownStateMixin = {
   },
 
   bindRootCloseHandlers() {
-    let doc = React.findDOMNode(this).ownerDocument;
+    let doc = domUtils.ownerDocument(React.findDOMNode(this));
```

Comment:
I don't see a problem with adding React there. The amount of code it DRYs up is enough of an argument. This whole library also depends on React, so it's not introducing anything too crazy.